src/Properties.js

```diff
@@ -0,0 +1,28 @@
+var PropertiesMixin = {
+  setProperties: function (obj) {
+    for (var i in obj) {
+      if (this.feature.properties.hasOwnProperty(i)) {
+        this.feature.properties[i] = obj[i];
+      }
+    }
+  },
+  getProperty: function (field) {
+    return this.feature.properties[field];
+  },
+  addProperties: function (obj) {
+    for (var i in obj) {
+      if (!this.feature.properties.hasOwnProperty(i)) {
+        this.feature.properties[i] = obj[i];
+      }
+    }
+  },
+  deleteProperty: function (arr) {
+    for (var i = 0; i < arr.length; i++) {
+      if (this.feature.properties.hasOwnProperty(arr[i])) {
+        delete this.feature.properties[arr[i]];
```

Comment:
Particular example, when you work with WFST, is a set property to null. 
But with setProperties this function also unnecessary, all this this work can be done with
```javascript
setProperties({a: null});
```

I was wrong, sometimes programmer should can remove properties from wfs:Transaction request, and deleteProperties can help.
Please restore function to you first version, but it should be renamed to deleteProperties.